pages/api/admin/users/[id]/password.js

```diff
@@ -0,0 +1,20 @@
+import { handleRequests } from "@klaudsol/commons/lib/API";
+import { withSession } from "@klaudsol/commons/lib/Session";
+import { OK } from '@klaudsol/commons/lib/HttpStatuses';
+import InsufficientDataError from '@klaudsol/commons/errors/InsufficientDataError';
+import People from '@klaudsol/commons/models/People';
+
+export default withSession(handleRequests({ put }));
+
+async function put(req, res) {
+    const { id } = req.query;
```

Comment:
Kindly use assertUserCan at the beginning of all API endpoints.

https://github.com/klaudsol/klaudsol-commons/blob/main/src/lib/Permissions.js#L78